cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Install locally with pip
+
+```
+pip install cloudcomposerdiff
+```
+
+### Authorizing the application using gcloud
+
+Install [gcloud](https://cloud.google.com/sdk/gcloud)
+
+Select a user which has access to both composer environments to compare.
+
+Wit this user account, initialize gcloud:
+
+```
+gcloud init
+```
+
+Next, authorize the application to access cloud composer environments:
+
+```
+gcloud auth application-default login
+```
+
+### Use the appplication
+
+```
+cloudcomposerdiff \
+--env1_project_id <> \
+--env1_location <> \
+--env1_name <> \
+--env2_project_id <> \
+--env2_location <> \
+--env2_name <>
+```
+
+## Developing this application
+
```

Comment:
Add prereqs here like version of python

Reply:
Good point. 

I have added a prereqs section here as well.